src/components/layout/MainOrgLayout.tsx

```diff
@@ -1,6 +1,6 @@
 //TODO: Enable eslint rule and fix errors
 /* eslint-disable  @typescript-eslint/no-non-null-assertion */
-import { Content } from '@react-spectrum/view';
+import { Content } from '@adobe/react-spectrum';
```

Comment:
What is this change about? Just so I understand what's going on.

Reply:
For some reason the `Content` was imported from `@react-spectrum/view`. `view is a specific and different component from `Content`. I've imported spectrum components from `@adobe/react-spectrum` unless it's necessary with a specific path as with `tabs` component.